iidm/iidm-xml-converter/src/main/java/com/powsybl/iidm/xml/AbstractTransformerXml.java

```diff
@@ -19,7 +19,7 @@
 /**
  * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
  */
-abstract class AbstractTransformerXml<T extends Connectable, A extends IdentifiableAdder<A>> extends AbstractConnectableXml<T, A, Container<? extends Identifiable<?>>> {
+abstract class AbstractTransformerXml<T extends Connectable<? super T>, A extends IdentifiableAdder<T, A>> extends AbstractConnectableXml<T, A, Container<? extends Identifiable<?>>> {
```

Comment:
Same question as in `BranchAdder`

Reply:
This is no needed in fact, I'd better remove it. I left it in case we want to do something similar to TieLine, but do we really...?

Reply:
I don't think we will